### PR TITLE
Weather PR Review

### PR DESCRIPTION
**Pull request by udberg**
This is a review request for the weather app. This project showcases the application of asynchronous javascript to work with a weather API that collects weather data from different cities and countries across the world.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @udberg 👋

**You've done quite well here, good use of es6 syntax, well-written readme, and good idea using a loading spinner to await the weather data requests. 👌**

However, let's make a couple of changes.

- [x] Please include eslintrc.json in the project, it is required that the project be properly linted using the config rules stated [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).

- [x] OPTIONAL - Consider fetching the current geolocation of the user when the app is first accessed and get the weather from that location.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Linters
```
Comment (addod19) [inline .github/workflows/linter.yml]: Kindly use the right linter configuration file..... You can check this [link](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml) for the required file

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @udberg, congratulations on this project :+1: We are almost through, except that we need to use the right linter configuration as I have commented below:

**N/B**: We do not need the `.eslintrc.json` or `.stylelintrc.json` files if we have the `.github/workflows/linters.yml` file
linters


Also, once we have the `.eslintrc.json` file we do not need to make any changes to the files as we have done. Kindly check the required file [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)

Once this is resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:

### Hi @udberg      ! :wave: :wave:

#### Good job implementing some of the requested changes.  :clap: However, there are still changes required for you to make for this project to be approved. Kindly use the correct `linters.yml` file as requested by the previous TSE, also make sure to change the file name from `linter.yml` to `linters.yml`.

 
- - -

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Review by oloomoses — APPROVED** [review]
## Congratulations, Project Aproved! :tada: :muscle: 
**Hello @udberg,
Well done on making the changes suggested by the previous reviewers. You have passed all the requirements for this project :100:** 

Happy Codding and best wishes in your next project

_slack: Oloo Moses_

**Pull request merged**